modules/applications-create-using-cli-template.adoc

```diff
@@ -29,19 +31,24 @@ $ oc new-app -f examples/sample-app/application-template-stibuild.json
 When creating an application based on a template, use the
 `-p|--param` argument to set parameter values that are defined by the template:
 
+[source,terminal]
 ----
 $ oc new-app ruby-helloworld-sample \
     -p ADMIN_USERNAME=admin -p ADMIN_PASSWORD=mypassword
 ----
 
-You can store your parameters in a file, then use that file with
-`--param-file` when instantiating a template. If you want to read the
-parameters from standard input, use `--param-file=-`:
+You can store your parameters in a file, then use that file with `--param-file` when instantiating a template. If you want to read the parameters from standard input, use `--param-file=-`. The following is an example file called `helloworld.params`:
 
+[source,terminal]
 ----
-$ cat helloworld.params
 ADMIN_USERNAME=admin
 ADMIN_PASSWORD=mypassword
+----
+
+Reference the parameters in the file when instantiating a template:
+
+[source,terminal]
+----
 $ oc new-app ruby-helloworld-sample --param-file=helloworld.params
 $ cat helloworld.params | oc new-app ruby-helloworld-sample --param-file=-
```

Comment:
Same here. Can we split these two up?

Reply:
Thanks Cody. On review, I have removed the second command, since it does the same thing as the first in a different way. Resolved.